traffic_portal/app/src/common/modules/form/deliveryServiceStaticDnsEntry/form.deliveryServiceStaticDnsEntry.tpl.html

```diff
@@ -57,9 +57,12 @@
                 </div>
             </div>
             <div class="form-group" ng-class="{'has-error': hasError(dsStaticDnsEntryForm.address), 'has-feedback': hasError(dsStaticDnsEntryForm.address)}">
-                <label for="address" class="control-label col-md-2 col-sm-2 col-xs-12">Address *</label>
+                <label for="address" class="has-tooltip control-label col-md-2 col-sm-2 col-xs-12">Address *<div class="helptooltip">
+                    <div class="helptext">The Address Rules.<br>1. Type:A_RECORD, Address should be an IPv4 address. <br>2. Type: AAAA_RECORD, Address should be an IPv6 address. <br>3. Type: CNAME, Address must end with a trailing period if type:CNAME. Eg: cdn.test.com. is correct but cdn.test.com will not work</div>
```

Comment:
this description seems a little redundant:

`Type: CNAME, Address must end with a trailing period if type:CNAME` also it doesn't describe the valid dns part. how about:

`Type: CNAME, Address must be a valid DNS name with a trailing period`

not even sure you need the example tbh. 

also, that line is pretty long. want to break it up?

Reply:
Yeah, i'll change it.